### ISSUE TITLE
Upload strings and more security?

### ISSUE THREAD

**Issue by steelzeh** [willpowell8/LocalizationKit_iOS#16]
Could we get the option to upload .strings og .xliff files? .strings would be fine for at start.
It's a bit hard to hand some api key to your translator and ask him to translate through that.

Also if you have the api key you can fuck up the whole translation. Would be nice to be able to secure it with some password.

It takes 10 seconds on charles to find the api key, when the framework does the /languages call.
After that it's possible to change the translation without any other security.

**Comment by Felizolinha** [comment]
The security thing is a really big problem for production. The website definitely needs to ask for a login before changing anything for this to be viable in production :/

**Comment by willpowell8** [comment]
I agree. I have been updating this and will be adding this to next release.
